packages/server/lib/plugins/child/require_async_child.js

```diff
@@ -11,6 +11,6 @@ const util = require('../util')
 const ipc = util.wrapIpc(process)
 const run = require('./run_require_async_child')
 
-const { file, projectRoot } = require('minimist')(process.argv.slice(2))
+const { configFilePath, projectRoot } = require('minimist')(process.argv.slice(2))
```

Comment:
In the migration process, we use this to run the actual pluginsFile from 9.X. I would suggest we keep the word `file` to convey this notion.

Reply:
Sure thing. I didn't find where this was being used in the migration workflow. Mind sharing so I have reference?

Reply:
[`7bcae3a` (#20560)](https://github.com/cypress-io/cypress/pull/20560/commits/7bcae3aa2e90d71e350abf67b385ff1bcea3883f)

Reply:
https://github.com/cypress-io/cypress/blob/10.0-release/packages/data-context/src/actions/MigrationActions.ts#L42-L50